### DISCUSSION TITLE
Latest Frigate release - Label for ghcr.io

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#7004] [Ask A Question] [answered]
Hi, I was wondering if there was a way to put the latest release for Frigate into the Repository? 

The current repository tag I have is; ghcr.io/blakeblackshear/frigate:0.12.1

I tried doing ghcr.io/blakeblackshear/frigate:latest but that didn't work. Is there a way to get Unraid or similar to pull the latest release without changing the version number in the repository?

**Comment by NickM-27** [comment, marked as answer]
Yes, the tag is `stable`

**Comment by DrSpaldo** [comment]
Great, thanks @NickM-27 

Is there one for the betas when they are around?

**Comment by itkfilelor** [comment]
> Great, thanks @NickM-27
> 
> Is there one for the betas when they are around?

Yes, but to my knowledge they are by name only (no `edge` or `beta` tag) keep an eye on the discussions here for new betas.
